internal/pkg/agent/cmd/run.go

```diff
@@ -405,7 +374,11 @@ LOOP:
 
 func loadConfig(ctx context.Context, override cfgOverrider, runAsOtel bool) (*configuration.Configuration, error) {
 	if runAsOtel {
-		return configuration.DefaultConfiguration(), nil
+		defaultCfg := configuration.DefaultConfiguration()
+		// disable monitoring to avoid injection
```

Comment:
Appreciate the comment here but I didn't follow what you mean by "avoid injection".  Could you expand on this a bit in the comment, please?

Reply:
updated a comment. it's not really an issue now as we're passing `inputs: []`
added this just to be sure and have default reflecting use case a bit more